test/rpc/test_rpc_misc.py

```diff
@@ -200,6 +201,59 @@ def test_get_events_wrong_blockid_type():
     assert resp["error"]["code"] == PredefinedRpcErrorCode.INVALID_PARAMS.value
 
 
+@pytest.mark.usefixtures("run_devnet_in_background")
+@pytest.mark.parametrize(
+    "run_devnet_in_background",
+    [PREDEPLOY_ACCOUNT_CLI_ARGS],
+    indirect=True,
+)
```

Comment:
Can be simplified as

```suggestion
@devnet_in_background(*PREDEPLOY_ACCOUNT_CLI_ARGS)
```